packages/next/src/build/index.ts

```diff
@@ -3701,11 +3702,21 @@ export default async function build(
         })
       )
 
+      await shutdownPromise
+
+      await runAfterProductionBuild({
```

Comment:
We'd want this to run after `webpackBuild` / `turbopackBuild` instead of at the end of `next build` because there's a difference between `experimental-compile` and `experimental-generate` and right now it will be called in both of those, however the places where people want to hook into it (e.g. Sentry sourcemap uploading) only needs to run after compile (which falls under `experimental-compile`).

Reply:
If we're wanting this to be right after compilation is finished should we instead call it `afterCompilation`?